app/views/blogs/edit.html.erb

```diff
@@ -1,27 +1,27 @@
 <% if @blog %>
-<h1>Edit Blog</h1>
+  <h1>Edit Blog</h1>
 
-<%= form_with model: @blog do |form| %>
-  <div>
-    <%= form.label :title %><br>
-    <%= form.text_field :title %>
-    <% @blog.errors.full_messages_for(:title).each do |message| %>
-    <div><%= message %></div>
-    <% end %>
-  </div>
+  <%= form_with model: @blog do |form| %>
+    <div>
+      <%= form.label :title %><br>
+      <%= form.text_field :title %>
+      <% @blog.errors.full_messages_for(:title).each do |message| %>
+      <div><%= message %></div>
+      <% end %>
+    </div>
 
-  <div>
-    <%= form.label :body %><br>
-    <%= form.text_area :body %>
-    <% @blog.errors.full_messages_for(:body).each do |message| %>
-    <div><%= message %></div>
-    <% end %>
-  </div>
+    <div>
+      <%= form.label :body %><br>
+      <%= form.text_area :body %>
+      <% @blog.errors.full_messages_for(:body).each do |message| %>
+      <div><%= message %></div>
+      <% end %>
+    </div>
 
-  <div>
-    <%= form.submit %>
-  </div>
-<% end %>
-<% else %>
-<h4> you can't edit anyone's blog</h4> 
+    <div>
+      <%= form.submit %>
+    </div>
+  <% end %>
+  <% else %>
```

Comment:
check indentation
also use tabs not space, go through the blogvault coding guidelines again